app/Http/Controllers/BookingRequestController.php

```diff
@@ -351,4 +352,38 @@ public function filter(Request $request)
     }
 
 
+
+    /**
+     * Filter booking requests by given json payload for a specific user
+     *
+     * @param Request $request
+     * @return JsonResponse|Response
+     */
+    public function myFilter(Request $request)
```

Comment:
`myFilter` is a bit ambiguous for a function name. Could you change it to something more descriptive, so as to better differentiate it from the `filter` function.

Reply:
i figured using myFilter would be a good name for the filter that is used on a user looking at their own bookings but sure i'll change it

Reply:
Name updated